src/components/ConfirmationBox/ConfirmationBox.js

```diff
@@ -1,28 +1,35 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 
-const ConfirmationBox = props => {
-	return (
-		<div>
-			<div>{`Did you take ${props.item.name}?`}</div>
-			{props.children}
+class ConfirmationBox extends React.Component {
```

Comment:
As a comment, if you are using ```React.Component``` in a lot of places I think it can look neater to

```
import React, { Component } from 'react';
```

and then

```
class ConfirmationBox extends Component {
```

not a big deal, just a preference